### PR TITLE
Feature newsweek home

### PR DESCRIPTION
**Pull request by Div685**
## About The Project

This project clones The [Newsweek](https://www.newsweek.com/), an American weekly news magazine which uses majorly bootstrap for the styling of the website.

The project has involved the application of bootstrap classes to HTML elements to achieve the desirable styling on the original page.

## Built With
- HTML
- CSS
- Bootstrap 4
- Fontawesome

## Getting Started
- Clone this repo https://github.com/Lameck1/Using-Bootstrap
- To get a local copy up and running follow these simple steps.
Install dependencies:

```
npm install
```

or

```
yarn install
```

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+body {
+  background-color: #f4f5f6;
+  color: #333;
+  line-height: 18px;
```
Comment (arslanbisharat) [inline css/style.css]: This goal can be done with bootstrap.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+.bg-header {
+  background: #f72210;
+}
+
+.title .icon-n {
+  -webkit-flex: none;
+  flex: none;
+  display: block;
+  width: 29px;
+  height: 28px;
```
Comment (arslanbisharat) [inline css/style.css]: The goals can be achieved with bootstrap.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  text-decoration-color: #f72210;
+}
+
+.bg-header {
+  background: #f72210;
+}
+
+.title .icon-n {
+  -webkit-flex: none;
+  flex: none;
```
Comment (arslanbisharat) [inline css/style.css]: This goal can be done with bootstrap.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  display: block;
+  width: 29px;
+  height: 28px;
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+  margin-right: 10px;
+}
+
+.image-side {
+  width: 200px;
```
Comment (arslanbisharat) [inline css/style.css]: This goal can be done with bootstrap.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+.news-img {
+  background-size: cover;
+  height: 18rem;
+}
+
+.vs-text {
+  margin: 0 auto;
+  width: 32px;
+  height: 32px;
```
Comment (arslanbisharat) [inline css/style.css]: This goal can be done with bootstrap.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  background: black;
+  color: #f4f5f6;
+  font-family: "Roboto Condensed", sans-serif;
+  display: grid;
```
Comment (arslanbisharat) [inline css/style.css]: This goal can be done with bootstrap.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+  place-items: center;
+  z-index: 5;
+  position: relative;
+}
+
+.versus::before,
+.versus::after {
+  position: absolute;
+  display: block;
```
Comment (arslanbisharat) [inline css/style.css:61]: This goal can be done with bootstrap.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  width: 100%;
```
Comment (arslanbisharat) [inline css/style.css:62]: This goal can be done with bootstrap.

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+  content: "";
+  height: 1px;
```
Comment (arslanbisharat) [inline css/style.css:64]: This goal can be done with bootstrap.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+  z-index: 5;
+  position: relative;
+}
+
+.versus::before,
+.versus::after {
+  position: absolute;
+  display: block;
+  width: 100%;
+  content: "";
```
Comment (arslanbisharat) [inline css/style.css]: Why this empty is here?

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+.header-middle svg {
+  fill: white;
+}
+
+.img-overlay {
+  top: 15px;
+  left: 80px;
+}
+
+/* @media (min-width: 768px) {
```
Comment (arslanbisharat) [inline css/style.css]: Please remove the commented code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ]  In Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. [Please refer to the main page of bootstrap to learn more about bootstrap. ](https://getbootstrap.com/docs/4.5/getting-started/introduction/)

- [ ] Please add non-necessary files like vscode to gitignore

- [ ] Navbar should be sticky. But in all versions of screens, it disappears from your website. 

**Clone**

chrome-capture (54)

**Original**

chrome-capture (55)


- [x] Please add these two folders into one folder. This will make your repo more professional. 
chrome-capture (45)

- [x] Please have a look at the attachment below. The newsletter image has more width on the original page.

**Clone**

chrome-capture (44)

- [ ] There is a breakdown after each item on the original page. But your clone is missing. 

**Clone**

chrome-capture (47)

**Original** 

chrome-capture (46)

- [ ] On the original page, section **The Debate** is present before **More Stories** but on your clone is the opposite. 

- [ ] Footer on tablet and small screen modes are not aligned correctly. please work on it. 

**Clone**

chrome-capture (48)

**Original** 

chrome-capture (49)

- [ ] There should be space and breakdown after the subscribe section on the small screen. 

**Clone**

chrome-capture (51)

**Original**

chrome-capture (52)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: css/style.css (hunk 7 above)
Comment (Lameck1) [inline css/style.css]: Hello, please lead me to a resource that would teach me how to implement pseudo selection with bootstrap.
I can't seem to find anything about implementing ::before and ::after with pure bootstrap. I am also kindly asking you to investigate why we decided to implement pseudo selections with CSS. Lastly, the project instructions tell us to use bootstrap as much as possible, not on everything, there are some styling on the page that are not just achievable with pure bootstrap, i.e. some sort of calculated fixed positioning, I was hoping you get to understand this and don't force us to undo the entire CSS.

File: css/style.css (hunk 8 above)
Comment (Lameck1) [reply css/style.css:62]: Still having a problem with implementing pseudo selection with bootstrap, please lead us to a resource or at least give us an idea.

File: css/style.css (hunk 9 above)
Comment (Lameck1) [reply css/style.css:64]: These are pseudo selections properties, they are trying to achieve a specific styling not achievable by bootstrap. Please give us an idea of how to do this with bootstrap.

File: css/style.css (hunk 10 above)
Comment (Lameck1) [reply css/style.css]: The content is empty because we are trying to achieve only a background without any content.

File: css/style.css (hunk 11 above)
Comment (Lameck1) [reply css/style.css]: Sure, that will be removed.

File: css/style.css (hunk 5 above)
Comment (Lameck1) [reply css/style.css]: We will do our best in this area but the bootstrap implementation of it is not working well. Any ideas?

File: css/style.css (hunk 4 above)
Comment (Lameck1) [reply css/style.css]: Bootstrap is not giving us the desired width at this point, we will do our best though.

File: css/style.css (hunk 3 above)
Comment (Lameck1) [reply css/style.css]: I would like to beg you to let this slide. This is an implementation of the official Newsweek icon. We can do it with bootstrap but it doesn't look good and we are sure TSEs won't accept an ugly implementation.

File: css/style.css (hunk 2 above)
Comment (Lameck1) [reply css/style.css]: I would like to beg you to let this slide. This is an implementation of the official Newsweek icon. We can do it with bootstrap but it doesn't look good and we are sure TSEs won't accept an ugly implementation.

File: css/style.css (hunk 1 above)
Comment (Lameck1) [reply css/style.css]: Agreed, we will make changes here.

File: css/style.css (hunk 5 above)
Comment (arslanbisharat) [reply css/style.css:48]: THANK YOU LOOKING FORWARD

File: css/style.css (hunk 4 above)
Comment (arslanbisharat) [reply css/style.css]: Thank you

File: css/style.css (hunk 10 above)
Comment (arslanbisharat) [reply css/style.css]: okay

File: css/style.css (hunk 1 above)
Comment (arslanbisharat) [reply css/style.css]: Thank You

File: css/style.css (hunk 2 above)
Comment (arslanbisharat) [reply css/style.css]: Next, TSE will review it and tell you a reasonable answer.

File: css/style.css (hunk 3 above)
Comment (arslanbisharat) [reply css/style.css]: Let the next TSE have a look.

**Comment by arslanbisharat** [comment]
### Apologizing About My mistake. 
Hello Team,
All TSEs are human so, we make mistakes too. I would like to apologize to both of you to please forgive me for my mistake. I didn't notice the pseudo-classes and asked you to make changes in code. 
Regards,
Arslan Bisharat.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done an excellent job so far.__

- Your responsiveness is sweet :100: 
- You did well in fixing the changes recommended by the previous reviewer :+1: 
- All linter checks are green :green_circle: 

_Overall, the project is almost complete. Let's make this correction and submit another review request. Please check my points below and fix them as appropriate._

**[ REQUIRED CHANGES ]**

- [x] The height of the newsletter section is not the same as the original. Kindly fix it.

Original page:


Your clone:


- [x] The Newsweek logo is not centralized in the top navbar

Original page:
original

Your clone:
clone


**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

                                            ### LARGE SCREEN

- [ ] Navbar in the large screen is not as present on the original page. I have already told you to change it in our chat in slack and during our video call. Please work on it. If you have any questions please reach me out or have a 1:1 appointment with a full-time TSE for greater explanation. 

- [ ] Text in the **subscribe** section is not correctly aligned as present on the original page. Please work on it. 

**Clone**

chrome-capture (88)

**Original**

chrome-capture (89)

- [ ] Under the Newsweek logo there is a very small gap on the original page but in your clone, it has more gaps. 

**Clone**

chrome-capture (90)

**Original**

chrome-capture (91)

                                         ### MEDIUM SCREEN

- [ ] In your clone on the medium screen there is a gap between the navbar and other sections but on the original page, there is no gap. Please have a look below. 

 **Clone**

chrome-capture (92)

**Original**

chrome-capture (93)


                                            ### SMALL SCREEN

- [ ] In your clone on the medium screen there is a gap between the navbar and other sections but on the original page, there is no gap. Please have a look below. 

 **Clone**

chrome-capture (92)

**ORIGINAL**

chrome-capture (94)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by arslanbisharat — APPROVED** [review]
### Congratulations Approved 🎉

Your project is approved with the latest changes you made. 
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

 **I have given some suggestions below. I would highly recommend you to do that.** 

**Optional** Please update the image in the readme. 

**OPTIONAL** If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**